doc/minutes/2022-12-01.md

```diff
@@ -0,0 +1,57 @@
+---
+title:  "First meeting YOS Award"
+date:   "30-11-2022"
+editor_options:
+  markdown:
+    canonical: false
+---
+
+**Attendees:** 
+
+-   Daniel Morillo Cuadrado
+
+-   Roberto Mediavilla Torres
+
+-   Cristina Rodríguez-Prada
+
+-   Lea Francia
+
+-   Jesus Sierralaya
+
+**Scribe:** Jesus Sierralaya
+
+**Schedule:** 11:00 - 11:30 
+
+# Action points
+
+## Project Proposals
+
+#### Debate 
+
+-   Cloud data management system and federated analysis with Opal
+    
+    -   We already have a physical server
+
+-   Reproducible research best practices with git/GitHub and literate computing
+
+-   We can use Synchros as example
+
+##   Prize assignation
+
+    -   Pay a freelancer to maintain the server
+
+    -   General communication
+
+    -   Staff training
+
+#### Agreements 
+
+-   **Approved**: Make a combination of both proposals
+
+-   **Approved**: Use the money for staff training
+
+## **Task summary:**
```

Comment:
Include the tasks also in the corresponding action point (see template for format)